util/codecov-ci.sh

```diff
@@ -17,26 +18,59 @@ fi
 
 BUILD="$1"
 if [ -n "$BUILD" ]; then
-  if [ ! -e "$BUILD" ]; then
-    mkdir -p "$BUILD"
-  fi
+  mkdir -p "$BUILD/default"
+  mkdir -p "$BUILD/no_asm"
+  mkdir -p "$BUILD/html"
   BUILD=$(readlink -f "$BUILD")
+  BUILD_DEFAULT=$(readlink -f "$BUILD/default")
+  BUILD_NO_ASM=$(readlink -f "$BUILD/no_asm")
+  BUILD_HTML=$(readlink -f "$BUILD/html")
 else
   echo "Must specify a build directory."
   exit 1
 fi
 
-cmake -DGCOV=1 -DDISABLE_PERL=1 -DBUILD_TESTING=1 -DBUILD_LIBSSL=1 -DCMAKE_BUILD_TYPE=Debug -S "${SRC}" -B "${BUILD}"
-cmake --build "${BUILD}" --target all_tests
-cmake --build "${BUILD}" --target run_tests
+LCOV_PARAMS=()
+LCOV_PARAMS+=(--exclude '*/third_party/*')
+LCOV_PARAMS+=(--exclude '*/tool/*')
+LCOV_PARAMS+=(--exclude '*_test.*')
+LCOV_PARAMS+=(--exclude '*/test_*')
+LCOV_PARAMS+=(--exclude '*_test_*')
+LCOV_PARAMS+=(--exclude '*/gtest_*')
+LCOV_PARAMS+=(--exclude '*/wycheproof_*')
```

Comment:
Can we initialize this list upfront on line 33? (You can make it multi-line too). Easier to read / less verbose

Reply:
I think I prefer the multi-line approach here. It's easier to read, and easier to add/modify if needed.